internal/app/init/pkg/system/runner/containerd/containerd_test.go

```diff
@@ -68,7 +68,7 @@ func (suite *ContainerdSuite) SetupSuite() {
 	args := &runner.Args{
 		ID: "containerd",
 		ProcessArgs: []string{
-			"/rootfs/bin/containerd",
```

Comment:
You will see a number of similar changes in this PR. The reason is because we used to add the rootfs to `/rootfs`, but since `containerd` is now linked against `libseccomp` we need to place everything at `/` to make sure that the  search paths are good.